adminSiteClient/ImagesIndexPage.tsx

```diff
@@ -152,6 +160,33 @@ function UserSelect({
     )
 }
 
+// when updatedAt changes, the image will reload the src
+// but it looks like sometimes cloudflare doesn't update in time :(
```

Comment:
This only works _sometimes_, so maybe it's not worth it and we just let authors know that it can take up to a minute for the image to update after refreshing the page. Not sure.

Reply:
If it doesn't work reliably, I'd add the disclaimer to the page. I'm not sure if I understand the intended functionality. If we want to refresh the image every time the `updatedAt` changes, I think [we could pass it as a `key` instead](https://react.dev/learn/preserving-and-resetting-state#option-2-resetting-state-with-a-key).

Reply:
Updating the key was the first thing I tried, but it wouldn't actually make a new request, and then when I tried forcing it to re-render, it would load the cached version, so this is the only way I found to make it work. (Though it also sometimes hits the CF cache 😫)

I think I'll just keep it as it currently is and see what the authors make of it. I'll probably have to explain in slack that sometimes the preview doesn't update and that they need to click on the image to see what the actual current image is. Seems.. fine 🤷